### DISCUSSION TITLE
Gas Respirators Toggle Which Type

### DISCUSSION THREAD

**Discussion by jimmyG1** [littlesera/LSCG#198] [Ideas]
wood be cool if you can pick what gas is being put into the mask if it says all 3 of thim can go into the mask

**Comment by jimmyG1** [comment]
or even just 2 of thim

**Comment by jimmyG1** [comment]
like in the settings you can put a color attached to a gas like pink for the horny gas and it can be a setting on what gas is going into the mask yes i know this might not be possible

**Comment by littlesera** [comment]
The effects of the mask (+injectors/drinks) are all handled on the receiver end. This allows for an LSCG user to still be drugged by someone not using the mod. Enhancements to the drugs that are configured on the sender and acknowledged by the receiver is not impossible, but more work and probably not for a while

**Comment by jimmyG1** [comment]
ok well at lest its not impossible i fully believed it was

**Comment by jimmyG1** [comment]
i have an idea on how it can be done after you have set the gas color in settings if a idam have all gas on it like how it works with the individual ones thin you can put in something like /gas horny and win you turn on the glow it will be the color for horny gas and that gas running the command 2 times before trying it on will just run the first color but all the gas thay you picked
